app/serializers/user_serializer.rb

```diff
@@ -31,7 +31,10 @@ def va_profile
   end
 
   def services
-    # TODO: build the available services based on available ID's
-    []
+    %w(facilities hca edu-benefits).tap do |service_list|
+      service_list += %w(rx messaging) if object.can_access_mhv?
```

Comment:
That makes sense. Anybody have thoughts on where those constants ought to be defined?